### PR TITLE
Fixing the Rectangle.contains() function

### PR DESCRIPTION
**Pull request by MuriloMir**
There was a typo, I corrected it and I rewrote the line in a way that I think is more elegant since it follows the same way of writing in math where you just write a <= x < b.

### CODE REVIEW

File: color.d [hunk 1]
```diff
@@ -1512,7 +1512,7 @@ struct Rectangle {
 
 	/// ditto
 	bool contains(in Point p) {
-		return (p.x >= left && p.y < right && p.y >= top && p.y < bottom);
```
Comment (adamdruppe) [inline color.d]: I don't like how you reorganized things, all that needed to change was the y to an x. I made the change on my local thing

**Comment by adamdruppe** [comment]
It was a simple typo and your diff was too big for it so I made the change myself https://github.com/adamdruppe/arsd/commit/6ab8c638a1f3fbdcd2259ae16f4d6e2e45db94f8

**Comment by adamdruppe** [comment]
So to be clear, I see the point of your other change and it is kinda nice, maybe if I was used to it I'd like it, but I have written contains things in this order for a lot of years and am just really used to it.

**Comment by MuriloMir** [comment]
Alright, I understand.

**Pull request closed without merging**